manifest/var/www/viewer/dmarcts-report-viewer-config.php

```diff
@@ -14,5 +14,6 @@
 
 $default_hostlookup = 1;  # 1=on 0=off (on is old behaviour )
 $default_sort = 1;  # 1=ASCdening 0=DESCending (ASCending is default behaviour )
+$imapignoreerror = 1;
```

Comment:
Uhm, this variable is for the parser and not for the viewer? The viewer is not talking to IMAP AFAIK? In the report parser conf file this variable is described and set to 0 by default (maybe debatable)